app/cafe.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def __str__(self) -> str:
+        return self.name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        check = 0
+        for vaccine in visitor:
+            if vaccine == "vaccine":
+                check += 1
+        if check == 0:
+            raise NotVaccinatedError(
+                "All friends should be vaccinated"

```

Comment:
Therefore this should tell us something like `You are not vaccinated. Please do vaccine`. Or somethin else. Or `Vaccination is not found, make one`